src/EditorFeatures/CSharp/Formatting/Indentation/CSharpIndentationService.Indenter.cs

```diff
@@ -162,7 +163,7 @@ internal class Indenter : AbstractIndenter
                 if (token.IsSemicolonOfEmbeddedStatement() ||
                     token.IsCloseBraceOfEmbeddedBlock())
                 {
-                    Contract.Requires(
+                    Debug.Assert(
```

Comment:
this file itself confuses me.  We had Contract.Assert and Contract.Requires.  And both mapped to Debug.Assert.  Does that mean Contract itself had two methods that were identical just with different names?  Yikes...

Reply:
Hence my desire to remove the confusion!